lib/sort_by_length.rb

```diff
@@ -1,7 +1,41 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: n
+# Space complexity: n * m ? because of variable 'word'
 def sort_by_length(my_sentence)
```

Comment:
👍 However the time complexity is O(n^2) 

Do you need to create a new `sentence_array`?  Couldn't you just sort the original array directly?